pkg/verifier/notation/notation.go

```diff
@@ -234,7 +236,7 @@ func normalizeVerificationCertsStores(conf *NotationPluginVerifierConfig) error
 		}
 	}
 	if isCertStoresByType && isLegacyCertStore {
-		return re.ErrorCodeConfigInvalid.NewError(re.Verifier, conf.Name, re.EmptyLink, nil, "both old VerificationCertStores and new VerificationCertStores are provided, please provide only one", re.HideStackTrace)
+		return re.ErrorCodeConfigInvalid.WithDetail("VerificationCertStores is misconfigured with both legacy and new format").WithRemediation("Please provide only one format for VerificationCertStores. Refer to Notation verifier configuration: https://ratify.dev/docs/next/plugins/verifier/notation#configuration")
```

Comment:
should the link be "https://ratify.dev/docs/plugins/verifier/notation#configuration" instead of next?

Reply:
Thanks for catching!